### PR TITLE
privacy info (draft)

### PR DESCRIPTION
**Pull request by js-xc**

### CODE REVIEW

File: content/en/xtc/privacy/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    * Last Name
+    * Email
+* **Optional:**
+    * Phone Number (for monitoring notifications via text message)
+* {{< TODO / >}}usage data, which is probably stored for audit log etc.? 
+
+XTC does _not_ store any of your contractual information or payment data.
+
+### Data Deletion
+XTC does not do automated account management (i.e. deactivation of accounts).
```
Comment (rschwietzke) [inline content/en/xtc/privacy/_index.md:41]: Accounts don't expire currently, only open and unaccepted invocations will (soon) expire.

File: content/en/xtc/privacy/_index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< TODO / >}}How long is account data/test data stored? Which can I delete myself, how, and when will this take effect?
```
Comment (rschwietzke) [inline content/en/xtc/privacy/_index.md:43]: There is no data retention for load test data. As long as an org or project exists, this data is retained. Customers have to delete this data themselves if needed.

Accounts cannot be deleted at the moment. A customer can write to support@xceptance.com and ask for system-wide deletion.

**Review by rschwietzke — CHANGES_REQUESTED** [review]

File: content/en/xtc/privacy/_index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+description: How XTC processes your personal data.
+---
+
+You can find all the details about the processing of personal data on Xceptance websites in the [Xceptance Privacy Statement](https://www.xceptance.com/en/contact/privacy.html). This is the condensed version of how XTC is handling personal data:
+
+### Cookie Usage
+We use only necessary cookies to make our site work. Necessary cookies enable core functionality such as security, network management, and accessibility. You cannot disable these without breaking the functionality.
+
+### Data Storage (Hosting)
+All resources are hosted internally at _xtc.xceptance.com_.
```
[the remark this answers is not in the document]
Comment (js-xc) [reply content/en/xtc/privacy/_index.md:17]: That's true, but in that case, please explain what we are trying to say by "No external to xtc.xceptance.com hosting of resources" (see ticket).

[the remark this answers is not in the document]
Comment (rschwietzke) [reply content/en/xtc/privacy/_index.md:17]: Data is not hosted by any third party and does not leave the scope of XTC. Data storage also does not use any Google services.

File: content/en/xtc/privacy/_index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+All resources are hosted at _xtc.xceptance.com_. 
+
+Data is stored on hardware provided by Google Cloud, but not shared with Google. 
+
+### Data Privacy (Analytics/Tracking)
+XTC includes no external analytics services or tracking.
+
+None of your data generated by or entered in XTC will be shared with third parties.
+
+In case you chose to use an [external login provider]({{< relref "../basics/040-user-accounts/#external-login-providers-sso" >}}) for authentication you will exchange data with your login provider (Google or Microsoft) on external servers.
```
Comment (rschwietzke) [inline content/en/xtc/privacy/_index.md:26]: Comma: ...authentication , you will...
Remove: on external servers

File: content/en/xtc/privacy/_index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+### Data Processing (Account Data)
+In order to use XTC, you need to create an account. The information and personal data you share with us will be utilized solely for the use of XTC services. This information is: 
+* **Required:**
+    * First Name
+    * Last Name
+    * Email
+* **Optional:**
+    * Phone Number (for monitoring notifications via text message)
+
+XTC does _not_ store any of your contractual information or payment data.
```
Comment (rschwietzke) [inline content/en/xtc/privacy/_index.md:37]: XTC does not store any of contractual information or payment data.

File: content/en/xtc/privacy/_index.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+### Data Deletion
+There is no automated data retention for load test data in XTC. As long as an organization or project exists, this data is retained. You can delete this data yourself if needed.
+
+XTC does not do automated account management (i.e. deactivation/deletion of inactive accounts). If you need to delete your account, you can write an email to support@xceptance.com to ask for a system-wide deletion. Please note that audit logs will retain name and email address, even when the account has been deleted, for 180 days. After that, it is anonymized, e.g. John Doe -> J\*\*\* D\*\*\* and john.doe@anydomain.net -> john@\*\*\*.net. This data is kept for up to 5 years.
```
Comment (rschwietzke) [inline content/en/xtc/privacy/_index.md:42]: XTC does not perform automated account management

Please note, that audit logs will retain the name and email address for 180 days, even when your account has been deleted.

**Review by rschwietzke — CHANGES_REQUESTED** [review]

File: content/en/xtc/privacy/_index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+### Cookie Usage
+We use only necessary cookies to make our site work. Necessary cookies enable core functionality such as security, network management, and accessibility. You cannot disable these without breaking the functionality.
+
+### Data Storage (Hosting)
+All resources are hosted at _xtc.xceptance.com_. 
+
+Data is stored on hardware provided by Google Cloud, but not shared with Google. 
+
+### Data Privacy (Analytics/Tracking)
+XTC includes no external analytics services or tracking.
```
[the remark this answers is not in the document]
Comment (js-xc) [reply content/en/xtc/privacy/_index.md:22]: See below: "The information and personal data you share with us will be utilized solely for the use of XTC services." Is this about the same as "no internal use" or do you want it more explicit?

**Review by rschwietzke — APPROVED** [review]

**Pull request merged**